src/storage.h

```diff
@@ -66,97 +61,68 @@ typedef enum {
 
 // storage data structure
 typedef struct __attribute__((packed)) {
-        // Any 'freeToUse' areas ensure the compatibility to origin EEPROM layout and can be used by new value.
```

Comment:
Changing all this means that saved configurations will not work when updating firmware and Erase Flash needs to be used, correct?

Reply:
yes right.
i did talked about a 4.0 release with fiendie. There is the idea of postpone the new storage/config system to a 4.1 release.
If so, i would move this PR also to a 4.1 release, so there is only one point where all users have to do a flash erase and not two times.